meinberlin/apps/contrib/templates/meinberlin_contrib/item_detail.html

```diff
@@ -116,14 +116,11 @@ <h1 class="item-detail__title">{{ object.name }}</h1>
                 </div>
             </div>
         </div>
-    </article>
```

Comment:
Was the removal here on purpose? If yes, probably also need to delete opening tag?